homeassistant/components/clicksend/notify.py

```diff
@@ -33,7 +36,7 @@
 
 PLATFORM_SCHEMA = vol.Schema(
     vol.All(
-        PLATFORM_SCHEMA.extend(
+        NOTIFY_PLATFORM_SCHEMA.extend(
```

Comment:
I'm not sure why there is a `vol.All` here in clicksend.
Maybe it should be `PLATFORM_SCHEMA = NOTIFY_PLATFORM_SCHEMA.extend...`

Reply:
Probably some leftover from some validation that has since been removed but didn't remove the `vol.All`

Reply:
Updated...